old_api/__init__.py

```diff
@@ -0,0 +1,66 @@
+import hashlib
+from datetime import datetime, timezone
+
+import requests
+from django.conf import settings
+
+from old_api.schema import FileList, FileMetadata
+
+
+session = requests.Session()
+session.headers["Authorization"] = settings.AIRLOCK_API_TOKEN
```

Comment:
I've found that setting things like this at import time can make testing awkward sometimes. Might not be an issue if this is short-lived code though.

Reply:
That's a good point, I'll move it to the two places its used.